src/videojs.markers.js

```diff
@@ -287,7 +287,12 @@
          if (newMarkerIndex != currentMarkerIndex) {
             // trigger event
             if (newMarkerIndex != -1 && options.onMarkerReached) {
-              options.onMarkerReached(markersList[newMarkerIndex]);
+               var nextMarker = markersList[newMarkerIndex];
+               // do not auto trigger when user jumps over marker(s)
+               var triggerAccuracy = player.duration() * 0.01;
```

Comment:
Because 'timeupdate' event doesn’t have constant frequency I didn’t want to hard code some 'empirically determined' constant. It was a safe bet. But probably 'triggerAccuracy' variable should have lower value.